nest/02.nest.md

```diff
@@ -92,6 +111,16 @@ To see the list of all installed packages, use the command `nest list`.
 By default, this command won't print the dependencies of the installed packages.
 To do so, add the flag `--with-deps`.
 
+Example of output:
```

Comment:
I would go for the simple yet popular `Example`, without the `of output`. It's more coherent with the other examples and general what people may see in similar getting started.

What do you think?